resources/zib/zib-Burnwound.xml

```diff
@@ -0,0 +1,173 @@
+  </mapping>
+  <kind value="resource" />
+  <abstract value="true" />
+  <type value="Condition" />
+  <baseDefinition value="http://hl7.org/fhir/StructureDefinition/Condition" />
+  <derivation value="constraint" />
+  <differential>
+    <element id="Condition">
+      <path value="Condition" />
+      <short value="Burnwound" />
+      <alias value="Brandwond" />
+      <mapping>
+        <identity value="zib-burnwound-v3.4-2020EN" />
+        <map value="NL-CM:19.4.1" />
+        <comment value="BurnWound" />
+      </mapping>
+    </element>
+    <element id="Condition.category">
+      <path value="Condition.category" />
+      <slicing>
+        <discriminator>
+          <type value="value" />
+          <path value="$this" />
+        </discriminator>
+        <rules value="open" />
+      </slicing>
+      <min value="1" />
+    </element>
+    <element id="Condition.category:burnWoundCode">
+      <path value="Condition.category" />
+      <sliceName value="burnWoundCode" />
+      <min value="1" />
+      <max value="1" />
+      <patternCodeableConcept>
+        <coding>
+          <system value="http://snomed.info/sct" />
+          <code value="125666000" />
+        </coding>
+      </patternCodeableConcept>
+    </element>
+    <element id="Condition.code">
+      <path value="Condition.code" />
+      <short value="BurnType" />
+      <definition value="Indication of the kind of the burn wound, differentiated into the cause of the injury." />
+      <alias value="BrandwondSoort" />
+      <binding>
+        <strength value="extensible" />
+        <valueSet value="http://decor.nictiz.nl/fhir/ValueSet/2.16.840.1.113883.2.4.3.11.60.40.2.19.4.3--20200901000000" />
+      </binding>
+      <mapping>
+        <identity value="zib-burnwound-v3.4-2020EN" />
+        <map value="NL-CM:19.4.9" />
+        <comment value="BurnType" />
+      </mapping>
+    </element>
+    <element id="Condition.bodySite">
+      <path value="Condition.bodySite" />
+      <short value="Location / AnatomicalLocation" />
+      <definition value="The location of the burn wound on the body." />
+      <alias value="AnatomischeLocatie" />
+      <max value="1" />
+      <type>
+        <code value="CodeableConcept" />
+        <profile value="http://nictiz.nl/fhir/StructureDefinition/zib-AnatomicalLocation" />
+      </type>
+      <mapping>
+        <identity value="zib-burnwound-v3.4-2020EN" />
+        <map value="NL-CM:19.4.12" />
+        <comment value="AnatomicalLocation" />
+      </mapping>
+    </element>
+    <element id="Condition.bodySite.extension:laterality">
+      <path value="Condition.bodySite.extension" />
+      <sliceName value="laterality" />
+    </element>
+    <element id="Condition.bodySite.extension:laterality.value[x]">
+      <path value="Condition.bodySite.extension.value[x]" />
+      <definition value="Laterality adds information about body side to the anatomic location, e.g. left" />

```

Comment:
Remove this definition, since we already fixed the typos within this definition in the zib-AnatomicalLocation data type.

Reply:
Fix